src/NUnitFramework/tests/Internal/TestExecutionContextTests.cs

```diff
@@ -43,28 +44,38 @@ namespace NUnit.Framework.Internal
     [TestFixture][Property("Question", "Why?")]
     public class TestExecutionContextTests
     {
-        TestExecutionContext _fixtureContext;
-        TestExecutionContext _setupContext;
-        ResultState _fixtureResult;
+        private TestExecutionContext _fixtureContext;
+        private TestExecutionContext _setupContext;
+        private ResultState _fixtureResult;
 
 #if !PORTABLE && !NETSTANDARD1_6
-        string originalDirectory;
-        IPrincipal originalPrincipal;
+        private string originalDirectory;
+        private IPrincipal originalPrincipal;
 #endif
 
-        DateTime _fixtureCreateTime = DateTime.UtcNow;
-        long _fixtureCreateTicks = Stopwatch.GetTimestamp();
+        private const string TempFileName = "NUnitTests.tmp";
+        private string _tempFilePath;
+        private DateTime _fixtureCreateTime = DateTime.UtcNow;
+        private long _fixtureCreateTicks = Stopwatch.GetTimestamp();
         
         [OneTimeSetUp]
         public void OneTimeSetUp()
         {
             _fixtureContext = TestExecutionContext.CurrentContext;
             _fixtureResult = _fixtureContext.CurrentResult.ResultState;
+            _tempFilePath = Path.Combine(TestContext.CurrentContext.WorkDirectory, "NUnitTests.tmp");
```

Comment:
Thanks. I had to do a bit of shuffling here to get things playing nice with PCL - must have missed that. 🙂